consult.el

```diff
@@ -236,10 +236,12 @@ See `consult--multi' for a description of the source data structure."
   "Regexp used to match file and line of grep output.")
 
 (defcustom consult-grep-args
-  "grep --null --line-buffered --color=never --ignore-case --line-number -I -r ."
+  '("grep" (consult--grep-generate-exclude-args-common)
+    "--null" "--line-buffered" "--color=never"
+    "--ignore-case" "--line-number" "-I" "-r" ".")
```

Comment:
You don't have to split all the strings here, since we us split-string-and-unquote.

Reply:
done